### ISSUE TITLE
email not shown

### ISSUE THREAD

**Issue by marcomagdy9** [emadunan/arab-deals#17]
why is the admin email not shown?
![image.png](https://images.zenhubusercontent.com/63962b6412960be401c14390/244d8769-6c7e-423e-ad62-f24e2b7cdc9d)

**Comment by marcomagdy9** [comment]
I think this is because the account entered through fb, is there a way to show any info from fb?

**Comment by emadunan** [comment]
> I think this is because the account entered through fb, is there a way to show any info from fb?

No, it's not exactly the case, but maybe because you didn't provide and confirm your email in your facebook account so how it could be returned?!
Try to provide email to your Facebook account and try again.
